sdk/sdk-api/src/main/java/mil/emp3/api/MirrorCache.java

```diff
@@ -34,23 +34,42 @@
 import mil.emp3.mirrorcache.spi.MirrorCacheClientProviderFactory;
 
```

Comment:
There is some documentation already but not in a centralized location yet. I'll add comments.